spec/yaml/swiftnav/sbp/signal.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Vlad Ungureanu <vvu@vdev.ro>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.signal
+description: Struct to represent the signal(constellation, band, PRN).
+stable: True
+public: True
+include:
+  - types.yaml
+definitions:
+
+ - sbp_signal:
```

Comment:
As this is not a message, I think it should be CamelCased for consistency, i.e. `SBPSignal`